File: app/src/main/java/com/resurrect/twentyweather/dataCuaca/Temperature.java
package com.resurrect.twentyweather.dataCuaca;

import com.google.gson.annotations.SerializedName;

/**
 * Created by root on 09/05/17.
 */

public class Temperature {
    @SerializedName("min")
    private float min;
    @SerializedName("max")
    private float max;

    public float getAvg(){
        return (min+max)/2;
    }
}

File: app/src/main/java/com/resurrect/twentyweather/dataCuaca/Temperature.kt
package com.resurrect.twentyweather.dataCuaca

import com.google.gson.annotations.SerializedName

/**
 * Created by root on 09/05/17.
 */

class Temperature {
    @SerializedName("min")
    private val min: Float = 0.toFloat()
    @SerializedName("max")
    private val max: Float = 0.toFloat()

    val avg: Float
        get() = (min + max) / 2
}
